lib/platform/github/index.ts

```diff
@@ -199,14 +219,23 @@ export async function initRepo({
     hostType: PlatformId.Github,
     url: defaults.endpoint,
   });
-  config.isGhe = URL.parse(defaults.endpoint).host !== 'api.github.com';
   config.renovateUsername = renovateUsername;
   [config.repositoryOwner, config.repositoryName] = repository.split('/');
   let repo: GhRepo;
   try {
+    let infoQuery = repoInfoQuery;
+    if (
+      config.isGhe &&
+      (!config.gheVersion ||
+        (config.gheVersion.startsWith('3.1') &&
+          semverLessThan(config.gheVersion, '3.1.8')) ||
+        semverLessThan(config.gheVersion, '3.0.16'))
```

Comment:
Allow versions released after Sept 23, 2021

Reply:
Let's link to the documents